CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ option(AMBER_ENABLE_SWIFTSHADER
   "Build using SwiftShader" ${AMBER_ENABLE_SWIFTSHADER})
 option(AMBER_ENABLE_RTTI
   "Build with runtime type information" OFF)
+option(AMBER_DISABLE_WERROR "Build without the -Werror flag" ${AMBER_DISABLE_WERROR})
```

Comment:
I think this will make both of these default to off: https://cmake.org/cmake/help/latest/command/option.html

Probably want `ON` instead of `${AMBER_DISABLE_WERROR}`.

Reply:
I was thinking we'd leave the same default behavior as before (with Werror enabled) so AMBER_DISABLE_WERROR should be off by default.

Reply:
Ah, good point. Double negatives are confusing. So, `OFF` by default means enable WError and WEverything